README.md

```diff
@@ -111,3 +111,25 @@ List your noticed symptoms and get immediate help on the medicines that could be
 
 * **Front-end:**  Web: AngularJS, Desktop App: Electron + AngularJS
 * **Back-end:** GoLang
+
+#  :trophy: Achievements
+  ### Jarvis-personal-assistant has been selected many times in various coding programmes like 
+  
+ ## KWoC-2019 (Kharagpur Winter of Code)
+ ![Kharagpur Winter of Code](https://miro.medium.com/max/957/1*gqkf7jD-6bxXTIHq4DaT_Q.png)
+
+The Kharagpur Winter of Code, often abbreviated as KWoC, is an annual five week long program where students new or unknown to free and open-source software contribute to coding projects registered in the program by experienced mentors or project owners and gain experience in contributing to open source software.The program is hosted by Kharagpur Open Source Society, an independent student group at IIT Kharagpur, but is open to students of any university
```

Comment:
I don't think such long descriptions are required describing the coding programme. Instead can provide the link to the website with the image. That is on clicking the image it can redirect to the website itself, so no need of description anymore.